openpype/settings/entities/schemas/projects_schema/schemas/schema_global_tools.json

```diff
@@ -97,6 +97,38 @@
                         ]
                     }
                 },
+                {
+                    "type": "list",
+                    "key": "open_workfile_tool_on_startup",
+                    "label": "Open workfile tool on launch",
+                    "is_group": true,
+                    "use_label_wrap": true,
+                    "object_type": {
+                        "type": "dict",
+                        "children": [
+                            {
+                                "type": "hosts-enum",
```

Comment:
Not all hosts support this. There should be explicit enum of hosts that are able to do that.

Reply:
Yes, you are right it should. Is there way to push list of allowed hosts which are supporting this at the moment? When you thing of the other feature `launch_last_workfile_at_start` it is also not supported by all hosts at the moment, right?

Reply:
> Is there way to push list of allowed hosts which are supporting this at the moment?

No but it would be handy on more places and I think it can be easily added to I can do it.

> it is also not supported by all hosts at the moment

True `standalonepublisher` does not have workfiles at all.

Reply:
> No, but it would be handy for more places and I think it can be easily added to I can do it.
Go ahead. And when it is done, I can merge it into this one ;)